run.py

```diff
@@ -1,2 +1,2 @@
 from eprihlaska import app
-app.run(debug=True)
+app.run(debug=True, host='0.0.0.0')
```

Comment:
Do we want to commit also host setting?

Reply:
Good catch!

Can you think of a good reason why that would not be a good idea?

Reply:
Not really. As I'm thinking about it, in production we do not run the app right from run.py, but rather use wsgi? ,  so we can keep the host setting there.

Reply:
That is indeed the case -- I do think we can easily keep the `host` setting here.